packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -802,8 +802,10 @@ const RelationsList = ({
             ...{
               apiData: {
                 id: relation.id,
-                documentId: relation.documentId,
-                locale: relation.locale,
+                documentId: relation.documentId
+                  ? relation.documentId
+                  : relation.apiData?.documentId || '',
```

Comment:
It's a bummer because the TypeScript isn't correct. I kind of prefer this syntax for this if it's all the same to you. 

```suggestion
                documentId: relation.documentId ?? relation.apiData?.documentId ?? ''
```